strategy/main.go

```diff
@@ -0,0 +1,81 @@
+package main
+
+import "fmt"
+
+// QuackStrategy interface
+type QuackStrategy interface {
```

Comment:
https://golang.org/doc/effective_go.html#interface-names

Reply:
Want me to name it `QuackStrateger`, `DisplayStrateger` and `FlyStrateger`? Sound a bit odd to me.

Reply:
It’s just a suggestion. 
Quacker. But that also sounds odd.

Reply:
Named it so to be in-sync with the explanation from the video. Let's maybe keep it that way?

Reply:
Yes yes. Sure.